### PR TITLE
Add plan review for MP OpenAPI 3.1

### PR DESCRIPTION
**Pull request by Emily-Jiang**

### CODE REVIEW

**Comment by dblevins** [comment]
As we don't use Hugo, let's rename _index.md and convert it to Asciidoc as say README.adoc

**Comment by Emily-Jiang** [comment]
> As we don't use Hugo, let's rename _index.md and convert it to Asciidoc as say README.adoc

Thanks @dblevins ! I have now converted this file to asciidoc.

File: openapi/3.1/README.adoc [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: "MicroProfile Open API 3.1 (under development)"
+date: 2022-06-28
+summary: "Release for MicroProfile 6.0"
+---
+MicroProfile Open API aims to provide a set of Java interfaces and programming models which allow Java developers to natively produce OpenAPI v3 documents from their JAX-RS applications.
```
Comment (ederks85) [inline openapi/3.1/README.adoc:6]: Can we still call it JAX-RS or better Jakarta RESTful Services?

Comment (Emily-Jiang) [reply openapi/3.1/README.adoc:6]: It is okay to still call JAX-RS as this acronym was mentioned many times in Jakarta RESTful Web Services.

**Review by ederks85 — APPROVED** [review]

**Pull request merged**